src/components/LeftSidebar/NewGroupConversation/NewGroupConversation.vue

```diff
@@ -496,15 +495,20 @@ it back */
 	height: 900px;
 }
 
+:deep(.modal-wrapper .modal-container) {
+			height: max-content;
+}
+
 .navigation {
+	position: sticky;
+    bottom: -1px;
 	display: flex;
 	justify-content: space-between;
 	flex: 0 0 40px;
-	height: 50px;
 	background-color: var(--color-main-background);
 	box-shadow: 0 -10px 5px var(--color-main-background);
 	z-index: 1;
-	width: 100%;
+	padding: 10px 20px;
```

Comment:
```suggestion
	padding: 0 20px 20px;
```
Check how that looks for you. I still think, that bottom offset should be 20, but reduced top also looks good to me. Also it's less noticable change because of `bottom: -1px`: 20 > 19 instead of 10 > 9

Reply:
Yeah, also good like this.